test/boilerplate_suggestors/annotations_remover_test.dart

```diff
@@ -0,0 +1,229 @@
+// Copyright 2020 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:over_react_codemod/src/boilerplate_suggestors/annotations_remover.dart';
+import 'package:over_react_codemod/src/boilerplate_suggestors/boilerplate_utilities.dart';
+import 'package:test/test.dart';
+
+import '../util.dart';
+
+main() {
+  group('AnnotationsRemover', () {
+    final testSuggestor = getSuggestorTester(AnnotationsRemover());
+
+    tearDown(() {
+      propsAndStateClassNamesConvertedToNewBoilerplate = {};
```

Comment:
I know this wasn't added in this PR, so it shouldn't be addressed here, but at some point we probably want to change it from a global variable and either pass it around to the relevant objects/functions or encapsulate it with the relevant utility methods. FYI @joebingham-wk 

In doing so, we'll 
- eliminate the risk of different areas of the code unintentionally interfering with each other's data (including in tests like this) 
- make it easier to reason about where the data is used
- decouple code that reads/writes to this map from the consuming code

Reply:
@greglittlefield-wf PR up to address this: https://github.com/Workiva/over_react_codemod/pull/79